modules/checking-cluster-resources-before-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-cluster-resources-before-initiating-limited-live-migration_{context}"]
+= Checking cluster resources before initiating the limited live migration
+
+The following procedure shows you how to check metrics for EgressIPs resources, EgressFirewalls resources, and multicast enabled namespaces. These resources are migrated to OVN-Kubernetes, however, they might behave differently after migration. You should check these resources before and after migration to ensure the intended behavior.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. As an {product-title} cluster administrator, check for EgressFirewall resources:
+
+.. Enter the following command to check for EgressFirewall resources:
++
+[source,terminal]
+----
+$ oc get egressnetworkpolicies.network.openshift.io -A
+----
++
+.Example output
++
+[source,terminal]
+----
+NAMESPACE    NAME                      AGE
+<namespace>  <example_egress_policy>   5d
+----
+
+.. You can check the intended behavior of an EgressFirewall resource by passing in the `-o yaml` flag. For example:
++
+[source,terminal]
+----
+$ oc get egressnetworkpolicy <example_egress_policy> -n <namespace> -o yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+apiVersion: network.openshift.io/v1
+kind: EgressNetworkPolicy
+metadata:
+  name: <example_egress_policy>
+  namespace: <namespace>
+spec:
+  egress:
+  - type: Allow
+    to:
+      cidrSelector: 0.0.0.0/0
+  - type: Deny
+    to:
+      cidrSelector: 10.0.0.0/8
+----
++
+Take note of the intended behavior of this resource, because it might change after migration.
+
+. Check your cluster for EgressIP resources:
```

Comment:
Here need to list the all egressip and multicast resources same with the egressnetworkpolicy
eg , list all namespace enable the egressip. 

```
$ oc get netnamespace -A | awk '$3 != ""'
NAME                                               NETID      EGRESS IPS
namespace1                                   14173093   ["10.0.158.173"]
namespace2                                    14173020   ["10.0.158.173"]
 
```

and find the namespaces enabled the multicast on openshift-sdn
```
oc get netnamespace -o json | jq -r '.items[] | select(.metadata.annotations."netnamespace.network.openshift.io/multicast-enabled" == "true") | .metadata.name'
```

Reply:
@zhaozhanqi PTAL at before and after resource checks:

Before:

After: